### ISSUE TITLE
Fuel loader eats stacked buckets

### ISSUE THREAD

**Issue by timmit99** [DarkPacks/SevTech-Ages#2642]
## Bug Report

Using an Inductive Logistics tank full of Galacticraft fuel you can shift click in a stack of 2+ buckets and it will fill the buckets with fuel (stack of 4 buckets -> 4 buckets of Fuel. Taking this stack of filled buckets to a fuel loader and shift clicking it into the inventory only empties one of the buckets and even though the entire stack of buckets was inserted, only one empty bucket is returned.

## Expected Behaviour

Either only move one bucket from the stack into the fuel loader, or empty ALL the buckets in the stack and return the full amount of buckets

## Possible Solution

Dont know enough about modded MC to know what to do =P

## Steps to Reproduce (for bugs)

Here is an imgur album showing the issue. https://imgur.com/a/DAqA7iQ

## Logs

Not a chash so no log here.

## Client Information

* Modpack Version: 3.0.6
* Java Version: 1.8.0
* Launcher Used: Twitch
* Memory Allocated: 16 GB
* Server/LAN/Single Player: Single Player
* Optifine Installed: no
* Shaders Enabled: no

**Comment by MJRLegends** [comment]
Stacking of buckets seems odd, by default vanilla doesnt stack buckets...

**Comment by timmit99** [comment]
Yeah, but with the 100+ mods buckets can stack inside some machines and i get stacks of fuel for my rockets so i dont have to carry 8 buckets from my filling station to my launchpad.

**Comment by justinrusso** [comment]
Yeah, this seems like an inductive logistics bug. That tank is very odd and also should not be giving stacks of filled buckets
